lib/run/summary.js

```diff
@@ -267,13 +267,28 @@ _.assign(RunSummary, {
     attachTimingTrackers (summary, emitter) {
         // mark the point when the run started
         // also mark the point when run completed and also store error if needed
-        emitter.on('start', function () { summary.run.timings.started = Date.now(); });
+        var x = 0;
+
+        emitter.on('start', function () {
+            summary.run.timings.started = Date.now();
+            summary.run.timings.iterationData = [];
+        });
         emitter.on('beforeDone', function () {
```

Comment:
Can we avoid calculating the iteration number here? At this part of the code, the system isn't decision maker of this data and can lead to a tonne of bugs down the line. Instead use the iteration number from the event itself.